pysb/tests/test_importers.py

```diff
@@ -34,15 +45,21 @@ def bngl_import_compare_simulations(bng_file, force=False,
         bng.execute()
         yfull2 = bng.read_simulation_results()
 
+    # Don't check trajectories on forced examples
+    if force:
+        return
+
     # Check all species trajectories are equal (within numerical tolerance)
-    for species in m.species:
-        print(species)
-        print(yfull1[species])
-        print(yfull2[species])
-        print(numpy.allclose(yfull1[species], yfull2[species], atol=1e-8,
-                             rtol=1e-8))
-        assert numpy.allclose(yfull1[species], yfull2[species], atol=1e-8,
-                              rtol=1e-8)
+    assert yfull1.dtype.names == yfull2.dtype.names
+    for species in yfull1.dtype.names:
+        logger.debug(species)
+        logger.debug(species)
+        logger.debug(yfull1[species])
+        logger.debug(yfull2[species])
+        logger.debug(numpy.allclose(yfull1[species], yfull2[species],
```

Comment:
Unsure if logging the result of `allclose` is useful, since the test passes/fails on that anyway.